packages/web3-net/src/rpc_method_wrappers.ts

```diff
@@ -0,0 +1,13 @@
+import { Web3Context } from 'web3-core';
+
+import * as rpcMethods from './rpc_methods';
+import { NetRPCApi } from './types';
+
+export const getId = async (web3Context: Web3Context<NetRPCApi>) =>
```

Comment:
Needs to include `returnType` functionality [e.g.](https://github.com/ChainSafe/web3.js/blob/ea41c556130558b37305767a39f25b4306dc0e50/packages/web3-eth/src/rpc_method_wrappers.ts#L43)